docs/QUICKSTART.md

```diff
@@ -0,0 +1,72 @@
+## Introduction
+
+This guide explains on how to setup the infrastructure to use and test the KUKSA Companion App.
+
+## Requirements
+
+- Android Device with API Level >= 27 (Android 8.1)
+- a running DataBroker
+- a running MockService (Optional)
+
+## Setup Databroker
+
+To use the Companion App a connection to a Databroker is required. This Databroker might run inside
+a car or on your local system for testing purposes. The steps to set up a local Databroker can be
+found in
+the [KUKSA.val Quickstart Guide](https://github.com/eclipse/kuksa.val/blob/master/doc/quickstart.md)
+
+## Setup Mock Service (Optional)
+
+This step is only required when connecting to a local running Databroker, which is not integrated
+into an actual vehicle.
+
+To use the Companion App the Databroker needs to be connected to the Mock Service.
+The Mock Service simulates the interaction with the car e.g. interactions with the different
+interactors of a car. The steps to set up a local Databroker can be found in the
+[KUKSA.val Quickstart Guide](https://github.com/eclipse/kuksa.val/blob/master/doc/quickstart.md)
+
+The Mock Service needs to be run with the
+following [mock.py](https://github.com/eclipse-kuksa/kuksa-android-companion/blob/main/mock/mock.py)
+
+## Configure Companion App
+
+When you have access to a Databroker, the next step is to setup the Companion App so it can connect
+to the Databroker.
+For this enter the "Settings" screen from inside the NavigationBar of the app.
+
+Here you have to enter the following connection information:
+
+- Host: The host on which the Databroker is running.
+- Port: The port on which the Databroker is running.
+  Make sure you used the same port, when starting the Databroker. The Apps default value is 55556.
```

Comment:
What do you think about changing to: "The default value expected by the App is 55556 while the Databroker by default starts at port 55555."

Reply:
Changed.